src/main/java/org/synyx/urlaubsverwaltung/sicknote/sicknote/SickNoteInteractionServiceImpl.java

```diff
@@ -95,6 +95,7 @@ public SickNote create(SickNote sickNote, Person applier, String comment) {
 
         sickNoteMailService.sendCreatedSickPerson(createdSickNote);
```

Comment:
hier wäre ein sendCreated**To**SickPerson leserlicher

Reply:
Ja, hab ich mal noch mit aufgenommen. Denke, das ist nicht verkehrt hier im Branch mitzumachen.